src/python/elements.cpp

```diff
@@ -216,6 +216,20 @@ void init_elements(py::module& m)
         )
     ;
 
+    py::class_<elements::Aperture>(mx, "Aperture")
```

Comment:
Surprised this does not cause trouble for us with our physical `"Aperture"` element. The names collide.

Reply:
Avoided because of `elements.mixin.` namespace we recently added to Python, vs. `elements.` namespace for the physical ones.

Renamed now to `PipeAperature` to avoid confusion.